packages/celotool/src/e2e-tests/sync_tests.ts

```diff
@@ -28,18 +36,54 @@ describe('sync tests', function(this: any) {
     await hooks.before()
     // Restart validator nodes.
     await hooks.restart()
+    const validatorKit = newKitFromWeb3(new Web3('http://localhost:8545'))
+    const groupPrivateKey = await getValidatorGroupPrivateKey(validatorKit, gethConfig.instances[0])
     const fullInstance = {
       name: 'full',
       validating: false,
       syncmode: 'full',
       lightserv: true,
-      port: 30311,
-      rpcport: 8553,
+      port: 30315,
+      wsport: 8557,
+      rpcport: 8559,
+      privateKey: groupPrivateKey.slice(2),
       peers: [8545],
     }
     await initAndStartGeth(hooks.gethBinaryPath, fullInstance)
-    const web3 = new Web3('http://localhost:8553')
+    const validators = await (await validatorKit._web3Contracts.getValidators()).methods
+      .getRegisteredValidators()
+      .call()
+    const web3 = new Web3('ws://localhost:8557')
```

Comment:
Nice!  We should probably favor ws over http regardless of needing `eth.subscribe()`.

Reply:
Totally agree, I'll create tech debt issue for this

Reply:
https://github.com/celo-org/celo-monorepo/issues/2616